config/templates.js

```diff
@@ -3,16 +3,16 @@ module.exports = {
     owner: '18f',
     repo: 'federalist-uswds-template',
     branch: 'master',
-    example: 'https://federalist-proxy.app.cloud.gov/site/18f/federalist-uswds-template/',
+    example: 'https://federalist-uswds-template.18f.gov',
     title: 'U.S. Web Design Standards',
     description: 'A flexible Jekyll implementation of the U.S. Web Design Standards',
     thumb: "/images/uswds.thumb.png",
   },
   'team': {
     owner: '18f',
-    repo: 'federalist-modern-team-template',
+    repo: 'https://federalist-modern-team-template.18f.gov',
```

Comment:
I think you changed the wrong property here. The new link should be down in `example:`, not in `repo:`.